taichi/program/callable.cpp

```diff
@@ -4,6 +4,10 @@
 namespace taichi {
 namespace lang {
 
+Callable::Callable() = default;
```

Comment:
Necessary to make MSVC happy, because `Callable` has a `std::unique_ptr<FrontendContext>` member, yet `FrontendContext` is forward declared.